src/test/regress/expected/columnar_citus_integration.out

```diff
@@ -0,0 +1,597 @@
+  SELECT strip_row_count FROM cstore.options WHERE regclass = '%s'::regclass;
+$cmd$);
+                            run_command_on_placements
+---------------------------------------------------------------------
+ (localhost,57637,20090008,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090008,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57637,20090009,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090009,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57637,20090010,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090010,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57637,20090011,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090011,f,"ERROR:  column ""strip_row_count"" does not exist")
+(8 rows)
+
+-- change setting
+SELECT alter_columnar_table_set('table_option', strip_row_count => 100);
+ERROR:  function alter_columnar_table_set(unknown, strip_row_count => integer) does not exist
+HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
+-- verify setting
+SELECT run_command_on_placements('table_option',$cmd$
+  SELECT strip_row_count FROM cstore.options WHERE regclass = '%s'::regclass;
+$cmd$);
+                            run_command_on_placements
+---------------------------------------------------------------------
+ (localhost,57637,20090008,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090008,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57637,20090009,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090009,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57637,20090010,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090010,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57637,20090011,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090011,f,"ERROR:  column ""strip_row_count"" does not exist")
+(8 rows)
+
+-- reset setting
+SELECT alter_columnar_table_reset('table_option', strip_row_count => true);
+ERROR:  function alter_columnar_table_reset(unknown, strip_row_count => boolean) does not exist
+HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
+-- verify setting
+SELECT run_command_on_placements('table_option',$cmd$
+  SELECT strip_row_count FROM cstore.options WHERE regclass = '%s'::regclass;
+$cmd$);
+                            run_command_on_placements
+---------------------------------------------------------------------
+ (localhost,57637,20090008,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57638,20090008,f,"ERROR:  column ""strip_row_count"" does not exist")
+ (localhost,57637,20090009,f,"ERROR:  column ""strip_row_count"" does not exist")
```

Comment:
strip -> stripe

also above

Reply:
good catch, something went wrong when I was working on that. I fixed that before, but somehow the test output also got reverted back. I just updated the output once I saw it was missing it. Missed that the type was back!

I have verified now that the test output looks good on all tests